crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
+    """
+    Read the names in ../names
+    :return: names_list
+    """
+    names_list = []
+    for name_file in glob.glob("./names/*"):
+        with open(name_file, 'r') as f:
+            first_line = f.readline()
+            first_name = first_line.split()[0]
+            names_list.append(first_name)
+    return names_list
+
+
+def greet(names):
+    greeting_list = ["Hello", "Hi", "Howdy"]
```

Comment:
```suggestion
    greeting_list = ["Hello", "Hi", "Howdy","Hola","Ciao"]
```
nice job, lovely to see you are going for the challenge!!!